source/js/Directions.js

```diff
@@ -0,0 +1,188 @@
+class Directions extends HTMLElement {
+    constructor() {
+      super();
+      let shadow = this.attachShadow({mode: 'open'});
+    }
+
+    set data(data) {
+        const style=`
+        .example {
+
+        }
+        `;
+
+    const styleElem = document.createElement('style');
+    styleElem.innerHTML=style;
+
+    let directionList = searchForKey(data, 'recipeInstructions');
+
+    const wrapper = document.createElement('article');
+    wrapper.classList.add('example');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8447291)
```suggestion
    wrapper.classList.add("example");
```